.devcontainer/on-create-command.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Immediately catch all errors
+set -eo pipefail
+
+# Uncomment for debugging
+# set -x
+# env
+
+git config --global --add safe.directory "*"
+
+CONSTRAINTS_URL=https://raw.githubusercontent.com/colcon/ci/main/constraints.txt
+CONSTRAINTS_FILE=/tmp/constraints.txt
+
+# Download constraints
+curl -sSLo $CONSTRAINTS_FILE $CONSTRAINTS_URL
+# Remove this package from constraints
+sed -i "/^$(python setup.py --name)@/d" $CONSTRAINTS_FILE
```

Comment:
I changed this line in colcon/ci#14 to support extensions that don't have a `setup.py`. You could copy the new logic here, or just drop in `colcon-clean` if you like.